R/sitrep.R

```diff
@@ -0,0 +1,68 @@
+#' laminr situation report
+#'
+#' Overview of the current status of the laminr package and its dependencies.
+#' Can be useful for debugging.
+#'
+#' @returns Prints details of the current laminr status
+#' @export
+#'
+#' @details
+#' Provides information that can be useful for debugging. To run the function
+#' when an error occurs set `options(error = laminr::laminr_sitrep)`. Note that
+#' this should be used with some caution as it will print the status whenever
+#' any error occurs.
+#'
+#' @examples
+#' laminr_sitrep()
+laminr_sitrep <- function() {
```

Comment:
Where does `laminr_sitrep` as a name come from? Can we have something more descriptive?

Reply:
I see; I didn't know the term other than what is available from `?laminr_sitrep` ^^

If you don't mind, I think `laminr_status` or `laminr_check` would be better